SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -26,17 +26,20 @@ bool FSpatialGDKEditor::GenerateSchema(bool bFullScan)
 		return false;
 	}
 
-	// Prompt the user to save packages/maps.
-	const bool bPromptUserToSave = true;
-	const bool bSaveMapPackages = true;
-	const bool bSaveContentPackages = true;
-	const bool bFastSave = false;
-	const bool bNotifyNoPackagesSaved = false;
-	const bool bCanBeDeclined = true;
-	if (!FEditorFileUtils::SaveDirtyPackages(bPromptUserToSave, bSaveMapPackages, bSaveContentPackages, bFastSave, bNotifyNoPackagesSaved, bCanBeDeclined))
+	// If this has been run from an open editor then prompt the user to save dirty packages and maps.
+	if (!IsRunningCommandlet())
 	{
-		// User hit cancel don't generate schema.
-		return false;
+		const bool bPromptUserToSave = true;
```

Comment:
Are none of these constants used below the call to `SaveDirtyPackages`?

Reply:
Probably just answered my own question: if it compiles, the answer is either "no" or "those variables were masking other ones". It's the second case I'm worried about. LGTM if you're confident this isn't the case.

Reply:
They are all used. These are all in the same scope though. (The red is because they have moved, not because they are gone)